### PR TITLE
Remove deprecated Theia extensions. Use VS Code built-in ones.

### PR DESCRIPTION
**Pull request by azatsarynnyy**
Signed-off-by: Artem Zatsarynnyi <azatsary@redhat.com>

### What does this PR do?
Removes the deprecated Theia extensions which will be deleted from upstream Theia.
Uses VS Code built-ins instead.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16133

### CODE REVIEW

**Comment by azatsarynnyy** [comment]
crw-ci-test

**Comment by azatsarynnyy** [comment]
crw-ci-test

File: generator/src/templates/assembly-package.mst [hunk 1]
```diff
@@ -18,16 +18,13 @@
     "@theia/console": "^{{ version }}",
     "@theia/core": "^{{ version }}",
     "@theia/debug": "^{{ version }}",
-    "@theia/debug-nodejs": "^{{ version }}",
```
Comment (benoitf) [inline generator/src/templates/assembly-package.mst]: just a note: should debug nodejs VS Code extension should be added somehow in the typescript che plugin ?

Comment (azatsarynnyy) [reply generator/src/templates/assembly-package.mst]: @benoitf I think yes, tslint and nodejs VS Code extension should be included into typescript che plugin
I'll propose a PR to plugin registry

**Comment by benoitf** [comment]
I think that yarn.lock should be bumped to theia 0.16.0 as well ?

File: generator/src/templates/assembly-package.mst (hunk 1 above)
Comment (azatsarynnyy) [reply generator/src/templates/assembly-package.mst]: provided PR https://github.com/eclipse/che-plugin-registry/pull/400

**Pull request merged**